src/frontend/components/app/action-header/action-header.tsx

```diff
@@ -73,9 +75,10 @@ const ActionHeader: React.FC<ActionHeaderProps> = (props) => {
       : [],
     params: { resourceId },
     handleClick: handleActionClick,
+    translateFunctions,
   })
 
-  const title = action ? action.label : resource.name
+  const title = action ? translateAction(action.label, resourceId) : resource.name
```

Comment:
Backend uses `ResourceDecorator#getResourceName` to set `resoure.name`. Previously it used `translateLabel` to translate resource name, now it's just id so what you assign here is not translated.

All `resource.name` usages should be changed to:
`translateLabel(resource.name, resource.name)`
or you can remove `resource.name` entirely and just use
`translateLabel(resource.id, resource.id)`
in the frontend